build-scripts/hack/upstream-images.yaml

```
@@ -49,11 +49,11 @@ sync:
   - source: quay.io/tigera/operator:v1.34.0
     target: ghcr.io/canonical/k8s-snap/tigera/operator:v1.34.0
     type: image
-  - source: quay.io/metallb/controller:v0.14.5
-    target: ghcr.io/canonical/k8s-snap/metallb/controller:v0.14.5
+  - source: quay.io/metallb/controller:v0.14.8
+    target: ghcr.io/canonical/k8s-snap/metallb/controller:v0.14.8
```

wait?  isn't it this:

```suggestion
    target: ghcr.io/canonical/metallb-controller:v0.14.8
```

---

Well I think this file is outdated anyway. It used to be that we pulled the images from quay and synced them to ghcr. But now we build the rocks. We don't want quay images to replace the rocks.